server/docs/authors-haven-api.yml

```diff
@@ -364,6 +364,32 @@ paths:
           description: Success. Email verification was successful
         500:
           description: Internal server errorcomponents     
+  /api/v1/users/profile:
```

Comment:
It appears this route is incorrect. The `routes/profiles.js` page requires a param to work. This returns a 404